Sources/Shell/Shell.swift

```diff
@@ -23,74 +27,74 @@ public enum Shell {
     /// - Returns: A string containing the command's standard output and standard error.
     /// - Throws: An error if the command execution fails
     @discardableResult public static func run(_ arguments: [String]) async throws -> String {
-        // TODO: It's probably a good idea to separate stdout and stderr streams.
+        logger.info("Executing: \(arguments.joined(separator: " "))")
 
-        // Log the command before execution
-        print("Executing: \(arguments.joined(separator: " "))")
-        
         let process = Process()
-        
+
```

Comment:
These formatting changes are a result of adding the `.swift-format` config file in #2 and me enabling format on save.